src/test/java/uk/gov/hmcts/reform/civil/bpmn/AddDefendantLitigationFriendTest.java

```diff
@@ -24,16 +24,19 @@ public AddDefendantLitigationFriendTest() {
     }
 
     @ParameterizedTest
-    @ValueSource(strings = {"true", "false"})
-    void shouldSuccessfullyCompleteNotifyClaim_whenCalled(Boolean rpaContinuousFeed) {
+    @CsvSource({"true, true", "true, false", "false, false", "false, true"})
+    void shouldSuccessfullyCompleteNotifyClaim_whenCalled(Boolean rpaContinuousFeed, boolean twoRespondents) {
         //assert process has started
         assertFalse(processInstance.isEnded());
 
         //assert message start event
         assertThat(getProcessDefinitionByMessage(MESSAGE_NAME).getKey()).isEqualTo(PROCESS_ID);
 
         VariableMap variables = Variables.createVariables();
-        variables.put("flowFlags", Map.of("RPA_CONTINUOUS_FEED", rpaContinuousFeed));
+        variables.put("flowFlags", Map.of(
+            "RPA_CONTINUOUS_FEED", rpaContinuousFeed,
+            "TWO_RESPONDENT_REPRESENTATIVES", twoRespondents)
```

Comment:
Not a major thing, but bear in mind for completeness that ONE RESPONDENT REPRESENTATIVE will actually be set to false in StateFlow Engine [here](https://github.com/hmcts/civil-service/blob/5fec28b67952f7aa1e1bb422b8e6cb57c642f37d/src/main/java/uk/gov/hmcts/reform/civil/service/flowstate/StateFlowEngine.java#L121)

But not a big issue as for now we dont use it in this dirgram!